httpretty/core.py

```diff
@@ -99,17 +99,16 @@
     if not PY3:
         old_sslwrap_simple = ssl.sslwrap_simple
     old_sslsocket = ssl.SSLSocket
+    old_sslcontext = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
```

Comment:
This does not work with Python 2.6:

    AttributeError: 'module' object has no attribute 'SSLContext'

I don't know if we care about Python 2.6 compatibility, though.

Reply:
Agreed, py2.6 should not block this security update